ui/src/shared/components/XYContainer.tsx

```diff
@@ -73,7 +66,14 @@ const XYContainer: FunctionComponent<Props> = ({
     storedYDomain,
     columnKeys.includes(yColumn) ? table.getColumn(yColumn, 'number') : []
   )
-  if (!xColumn || !yColumn) {
+
+  const isValidView =
```

Comment:
I tested this out with some line graphs that I had created before this change. It looks like this will invalidate all of them, since they don't have stored `xColumn` and `yColumn` settings:

<img width="1440" alt="Screen Shot 2019-06-04 at 2 27 33 PM" src="https://user-images.githubusercontent.com/638955/58915052-eda96080-86d4-11e9-98d5-628f75042106.png">

To fix this I think that we could fall back to using `chooseXColumn(table)` if a `xColumn` doesn't exist on the view properties, and similarly for the `yColumn`.

```
const xColumn = storedXColumn || chooseXColumn(table)
const yColumn = storedYColumn || chooseYColumn(table)
```

Reply:
We might also have to do this for the scatterplot too.

Reply:
thanks!